tensorflow/python/ops/ctc_ops.py

```diff
@@ -47,6 +47,18 @@ def ctc_loss(inputs, labels, sequence_length,
   max(labels.indices(labels.indices[:, 1] == b, 2))
     <= sequence_length(b) for all b.
   ```
+  
+  Useful Tips:
+  
+  The ctc_loss has as input an affine projection of RNN state:
+    inputs_t = W*h^L_{t} + b, for t = {1,...,max_time}
+  where h is the hidden state of last rnn calculated at timestep t.
+  
+  The `inputs` argument has size num_labels + 1 where +1 is reserved to
+  blank label, e.g., for a vocab = {a, b, c} the num_classes will be 4 
+  where {a: 0, b: 1, c: 2, blank: 3}.
+  
+  `ctc_loss` function already interleaves blank label between the target labels.
```

Comment:
Yes, I agree @ebrevdo, but only looking the ctc_ops.py documentation turns unclear how the blank label is inserted (and its's index).